Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DocumentPropertyEditorSettings.cpp

```diff
@@ -12,6 +12,11 @@
 
 namespace AzToolsFramework
 {
+    DocumentPropertyEditorSettings::DocumentPropertyEditorSettings()
+    {
+        m_shouldSaveSettings = false;
```

Comment:
this is set to false here, and to true inline in the header declaration. By convention, we use the header declaration

Reply:
That makes sense. I'll default it to false and set to true in the other ctor, then we can still have our `= default` ctor for this class.